packages/common/src/downloads/_internal/canUseExportImageFlow.ts

```diff
@@ -7,5 +7,9 @@ import { IHubEditableContent } from "../../core/types/IHubEditableContent";
  * @returns whether the export image flow can be used
  */
 export function canUseExportImageFlow(entity: IHubEditableContent): boolean {
-  return entity.type === "Image Service";
+  const { type, typeKeywords = [] } = entity;
+  // Tiled Imagery services cannot be downloaded. This typeKeyword check
+  // is one way to distinguish between tiled and non-tiled imagery services.
+  // TODO: Consider checking item.url instead so reference items are also excluded.
+  return type === "Image Service" && !typeKeywords.includes("Tiled Imagery");
```

Comment:
these two strings get repeated a good amount throughout the PR -- for future proofing, can these be put in a `const` or something more reusable? I get worried about magic strings

Reply:
Could we use these [item type enums](https://github.com/Esri/hub.js/blob/4400b4d6f1f09c7b0f8649732d32c69b258b2dd7/packages/common/src/types.ts#L446)?

Reply:
I can give that enum a go!